plugins/Marketplace/plugin.json

```diff
@@ -0,0 +1,7 @@
+{
+    "name": "Marketplace",
+    "version": "0.1.0",
+    "description": "Extend and expand the functionality of Piwik via the Marketplace by downloading plugins and themes.",
+    "theme": false,
+    "license": "GPL v3+"
+}
```

Comment:
Isn't that a core plugin and should therefor contain **no** `plugin.json`?
Btw. the plugin description should be added to `en.json`with key `PluginDescription`